src/pages/List.js

```diff
@@ -1,27 +1,28 @@
 import React, { useContext } from 'react';
 import NavTabs from '../components/NavTabs';
-import { ListContext } from '../listContext';
-import useListToken from '../useListToken';
-import { FirestoreCollection } from 'react-firestore';
 import Loading from '../components/Loading';
 import ErrorMessage from '../components/ErrorMessage';
-import HomePageButton from '../components/HomePageButton';
+import useListToken from '../useListToken';
+import { FirestoreCollection } from 'react-firestore';
+import { ListContext } from '../listContext';
 import dayjs from 'dayjs';
+import { useParams } from 'react-router-dom';
 import './List.css';
 
-const today = dayjs();
-
-function isLessThan24hrs(datePurchased) {
-  let purchaseDateCalc = dayjs(datePurchased);
-  return today.diff(purchaseDateCalc, 'hour') <= 24;
-}
-
 const List = props => {
   const { shoppingList, setShoppingList, addDatePurchased } = useContext(
     ListContext,
   );
-  const { token } = useListToken();
+  let { token: tokenFromUrlParams } = useParams();
```

Comment:
I'm concerned that we've introduced a 2nd source of truth for the user's token. I'd prefer we stick to only using the token in `localStorage` if at all possible. Thanks!